### PR TITLE
path params

### PR DESCRIPTION
**Pull request by vikispeaks**

### CODE REVIEW

File: react-router/path-params/README.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+In this example, `:id` is the path param, which can be any string value. When the user navigates to a URL that matches this route, React Router will extract the value of the id parameter from the URL and pass it as a prop to the UserDetails component.
+
+For example, if the user navigates to /users/123, React Router will pass the value "123" as a prop to the UserDetails component.
+
+To add path params to the existing project, we will modify the routes in our `App.tsx` file.
+
+Let's start by adding path params to the task details page so that we can display the details for a specific task.
+
+First, let's modify the TaskDetails component to accept a taskId prop:
```
Comment (reenas) [inline react-router/path-params/README.md:19]: @vikispeaks For clarity, include filename where the below snippet goes.

File: react-router/path-params/README.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+    </div>
+  );
+}
+
+export default TaskDetails;
+```
+
+Note that we're using the useParams hook from the react-router-dom package to access the taskId parameter from the URL. We're also passing the taskId prop to the component, which we'll use to display the task details.
+
+Next, let's modify the route for the TaskDetails component to include a taskId path param:
```
Comment (reenas) [inline react-router/path-params/README.md:48]: @vikispeaks Assuming this is in the same file, where will this line go in the whole file?

File: react-router/path-params/README.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+
+```js
+<Route path="/tasks/:taskId" component={TaskDetails} />
+```
+
+Now, when the user navigates to a URL like /tasks/123, React Router will pass the value "123" as the taskId prop to the TaskDetails component.
+
+Finally, let's modify the TaskList component to include links to the task details page that include the task ID as a path param:
```
Comment (reenas) [inline react-router/path-params/README.md:56]: @vikispeaks For clarity, include filename where the below snippet goes. If this is an earlier file, mention so if they need to delete old content and replace with the one below?

Comment (vikispeaks) [reply react-router/path-params/README.md]: Updated the file details. All of these are minor modifications to the existing file as stated in the content.

File: react-router/path-params/README.md (hunk 2 above)
Comment (vikispeaks) [reply react-router/path-params/README.md]: They have the same route defined in the previous lesson. They will be updating it here in the current one. This is just an updated line, not a new one. Have updated the file details for clarity.

File: react-router/path-params/README.md (hunk 1 above)
Comment (vikispeaks) [reply react-router/path-params/README.md]: Done.

**Pull request merged**